migration/20210604-reverse-update-datetime-columns.sql

```diff
@@ -2,20 +2,22 @@
 -- It should only be uncommented in the event that you need to deploy this fix
 -- via the build system. Otherwise, you should run the commands by hand.
 --
+-- These statements are grouped into the same batches as
+-- 20210405-update-datetime-columns.sql, and it may be possible to
+-- save time by running the batches in parallel, as described in that
+-- file.
+--
```

Comment:
Possibly also worth noting (or not) that these statements are all likely to run quite a bit faster than the forward migration. I think it's because these are shrinking the column storage to go from timestamp with tz back to timestamp without tz instead of having to allocate new space. I definitely saw faster ALTER completion times in the rollback we did for OE.